modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -15,6 +15,7 @@ export interface SchemaSettings {
   valid_elements?: string;
   valid_styles?: string | Record<string, string>;
   verify_html?: boolean;
+  retain_empty_block_inline_children?: boolean;
```

Comment:
```suggestion
  keep_empty_block_inline_children?: boolean;
```
Maybe? As looking elsewhere it seems we've used `keep` previously. That said given this enables the padding functionality maybe we should be using `padd` or similar instead?

Reply:
I think `padd_empty_block_inline_children` could work well and is the clearest representation of what is happening

Reply:
Switched over to `padd_empty_block_inline_children`